Public/Src/Utilities/UnitTests/Executables/InfiniteWaiter/Program.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// InfiniteWaiter.exe - Executable which launches and blocks forever.

```

Comment:
> InfiniteWaiter.exe  [](start = 2, length = 20)

Why creates a new exe instead of generalizing (and placing in a proper namespace) DummyWaiter?

Reply:
Too much drama for marginal gains... I'd have to deal with argument passing, etc. to identify the desired behavior. Considering this is a two line implementation, I'd prefer to leave it as is

---
In reply to: [339177541](https://github.com/microsoft/BuildXL/pull/1099#discussion_r339177541) [](ancestors = 339177541)